SwrveSDK/SDK/Track/Swrve.h

```diff
@@ -2,6 +2,7 @@
 #import "SwrveInterfaceOrientation.h"
 #import "SwrveReceiptProvider.h"
 #import "SwrveResourceManager.h"
+#import "SwrveBaseCampaign.h"
```

Comment:
This is for users who are directly copying the SDK into their projects. Currently with their Bridging-Header, they need to add the SwrveBaseCampaign to use MessageCenter. MessageCenter should have all it's objects readily available.